src/WorldStateCache.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
use the `json-fetch-cache` npm package instead, it does most of the heavy lifting on the caching